rdr_service/model/genomics.py

```diff
@@ -629,3 +629,24 @@ class GenomicIncident(Base):
 
 event.listen(GenomicIncident, 'before_insert', model_insert_listener)
 event.listen(GenomicIncident, 'before_update', model_update_listener)
+
+
+class GenomicCloudRequests(Base):
+    """
+    """
```

Comment:
Do you want to add a comment for this class?

Reply:
Yes I do.

Reply:
https://github.com/all-of-us/raw-data-repository/pull/2456/commits/2350c4383e93fba0bcec1843bfa8e33d4a8b7f26